react/src/components/LegacyFolderExplorer.tsx

```diff
@@ -164,26 +170,18 @@ const LegacyFolderExplorer: React.FC<LegacyFolderExplorerProps> = ({
         <Alert message={t('explorer.NoPermissions')} type="error" showIcon />
       ) : null}
       {vfolder_node ? (
-        xl ? (
-          <Splitter
-            style={{
-              gap: token.size,
-              alignSelf: 'stretch',
-            }}
-          >
-            <Splitter.Panel resizable={false}>
-              {legacyFolderExplorerPane}
-            </Splitter.Panel>
-            <Splitter.Panel defaultSize={500} min={300} max={'40%'}>
-              {vfolderDescription}
-            </Splitter.Panel>
-          </Splitter>
-        ) : (
-          <BAIFlex direction="column" align="stretch" gap={'md'}>
+        <Splitter
+          style={{
+            gap: token.size,
+            maxHeight: 'calc(100vh - 220px)',
+          }}
+          layout={xl ? 'horizontal' : 'vertical'}
+        >
+          <Splitter.Panel resizable={false} max={'60%'}>
             {legacyFolderExplorerPane}
-            {vfolderDescription}
-          </BAIFlex>
-        )
+          </Splitter.Panel>
+          <Splitter.Panel min={'40%'}>{vfolderDescription}</Splitter.Panel>
+        </Splitter>
```

Comment:
The previous layout using `defaultSize` looks better than the new layout.
```suggestion
        <Splitter
          style={{
            gap: token.size,
            maxHeight: 'calc(100vh - 220px)',
          }}
          layout={xl ? 'horizontal' : 'vertical'}
        >
          <Splitter.Panel resizable={false}>
            {legacyFolderExplorerPane}
          </Splitter.Panel>
          <Splitter.Panel defaultSize={500} min={300}>
            {vfolderDescription}
          </Splitter.Panel>
        </Splitter>
```